src/test/java/core/basesyntax/service/file/FileWriterTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.file;
+
+import core.basesyntax.service.file.impl.FileWriterImpl;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String PATH_TO_FILE_INPUT = "src/main/resources/data.csv";
+    private static final String PATH_TO_FILE_OUTPUT = "src/main/resources/output.csv";
+    private static final String PATH_TO_EXISTING_FILE_OUTPUT = "src/main/resources/existing.csv";
+    private static final String INCORRECT_FILE_PATH = "this?path/is/src/incorrect/main/file.csv";
+    private static final String TEXT = "Hello, World!";
+    private static FileWriter fileWriter;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void fileExist_isOk() throws IOException {
+        fileWriter.write(PATH_TO_EXISTING_FILE_OUTPUT, TEXT);
+        boolean exists = Files.exists(Path.of(PATH_TO_EXISTING_FILE_OUTPUT));
+        Assert.assertTrue(exists);
+        Files.delete(Path.of(PATH_TO_EXISTING_FILE_OUTPUT));
```

Comment:
I know it's better to add this to a method annotated with "after".

Reply:
it's being used just once, so it's ok